docs/release_notes.rst

```diff
@@ -18,7 +18,7 @@ This release introduces the following:
 - BIG-IP® vCMP® compatibility
 - Bug fixes
 
-See the `changelog <https://github.com/F5Networks/f5-openstack-lbaasv2-driver/compare/v8.0.7...v8.0.8>`_ for the full list of changes in this release.
```

Comment:
Per discussion with Rich during offline review, remove lines 28-30 in the "not supported" section.